website/docs/getting-started/install.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
```

Comment:
other pages in the docurous have index as well
you can check other pages they will be having 2